### PR TITLE
Get rid of Github Actions deprecations

### PR DESCRIPTION
**Pull request by NoResponseMate**

### CODE REVIEW

**Review by tucksaun — CHANGES_REQUESTED** [review]

File: commands/local_check_security.go [hunk 1]
```diff
@@ -72,9 +72,22 @@ a specific "composer.lock" file.`,
...
+            }
+
 			// Ran inside a Github action, export vulns
 			output, _ := security.Format(vulns, "raw_json")
-			terminal.Eprintf("::set-output name=vulns::%s", output)
+	        if _, err = f.WriteString("vulns=" + string(output) +  "\n"); err != nil {
+		        f.Close()
+		        return console.Exit(fmt.Sprintf("unable to write into github output: %s", err), 127)
+	        }
+	        if err = f.Close(); err != nil {
```
Comment (tucksaun) [inline commands/local_check_security.go:88]: I would move both `f.Close()` calls just after opening the file and using `defer`

File: commands/local_check_security.go [hunk 2]
```diff
@@ -72,9 +72,22 @@ a specific "composer.lock" file.`,
 		terminal.Stdout.Write(output)
 
 		if os.Getenv("GITHUB_WORKSPACE") != "" {
+			gOutFile := os.Getenv("GITHUB_OUTPUT");
+
+            f, err := os.OpenFile(gOutFile, os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0644)
```
Comment (tucksaun) [inline commands/local_check_security.go:77]: Golang community uses tabs for indentation, you can use `go fmt` to automatically format the file.

**Review by tucksaun — CHANGES_REQUESTED** [review]

**Comment by kevinpapst** [comment]
@NoResponseMate will that fix the deprecation warning `Warning: The `set-output` command is deprecated and will be disabled soon. Please upgrade to using Environment Files.`, which is triggered when `symfony security:check` is run as part of a GitHub workflow?
Bildschirmfoto 2022-11-13 um 17 30 47

**Comment by NoResponseMate** [comment]
@kevinpapst Yeah, that's the goal

**Comment by fabpot** [comment]
Thank you

**Pull request merged**